IO/deck.json

```diff
@@ -0,0 +1,354 @@
+                    "ord": 0,
+                    "rtl": false,
+                    "size": 20,
+                    "sticky": false
+                },
+                {
+                    "font": "Arial",
+                    "name": "Tył",
+                    "ord": 1,
+                    "rtl": false,
+                    "size": 20,
+                    "sticky": false
+                }
+            ],
+            "latexPost": "\\end{document}",
+            "latexPre": "\\documentclass[12pt]{article}\n\\special{papersize=3in,5in}\n\\usepackage[utf8]{inputenc}\n\\usepackage{amssymb,amsmath}\n\\pagestyle{empty}\n\\setlength{\\parindent}{0in}\n\\begin{document}\n",
+            "latexsvg": false,
+            "name": "INF",
+            "req": [
+                [
+                    0,
+                    "any",
+                    [
+                        0
+                    ]
+                ]
+            ],
+            "sortf": 0,
+            "tags": [
+                "wyklad1"
+            ],
+            "tmpls": [
+                {
+                    "afmt": "{{FrontSide}}\n\n<hr id=answer>\n\n{{Tył}}",
+                    "bafmt": "",
+                    "bfont": "",
+                    "bqfmt": "",
+                    "bsize": 0,
+                    "did": null,
+                    "name": "Karta 1",
+                    "ord": 0,
+                    "qfmt": "{{Przód}}"
+                }
+            ],
+            "type": 0
+        }
+    ],
+    "notes": [
+        {
+            "__type__": "Note",
+            "fields": [
+                "Podaj definicję&nbsp;<b>inżynierii oprogramowania</b>",
+                "systematyczne projektowanie i rozwój oprogramowania oraz zarządzanie procesem wytwarzania oprogramowania"
+            ],
+            "guid": "Ep%#<Ii)_c",
+            "note_model_uuid": "a2dd1c7a-7c66-11eb-9311-648099ca582d",
+            "tags": [
+                "wyklad1"
+            ]
+        },
+        {
+            "__type__": "Note",
+            "fields": [
+                "Podaj definicję <b>inżynierii</b>",
+                "Nauka o używaniu <i>metody naukowej</i>&nbsp;do projektowania i budowania maszyn, struktur budowlanych i innych rzeczy"
+            ],
+            "guid": "y?ABPdqR76",
+            "note_model_uuid": "a2dd1c7a-7c66-11eb-9311-648099ca582d",
+            "tags": [
+                "wyklad1"
+            ]
+        },
+        {
+            "__type__": "Note",
+            "fields": [
+                "Podaj definicję <b>metody naukowej</b>",
+                "Metoda naukowe to <b>empiryczna metoda</b>&nbsp;zdobywania wiedzy charakteryzująca sposób rozwoju nauki od około 17 wieku.<div><br></div><div>Składa się z etapów obserwacji i sceptycznej analizy obserwacji.&nbsp;</div><div><br></div><div>Etapy: obserwacja, sformułowanie hipotezy, eksperymentalne weryfikowanie następstw hipotezy, uszczegółowienie lub odrzucenie hipotezy.</div>"
```

Comment:
Nie wiem czy ta fiszka jest tu konieczna, a na pewno informacja o tym że charakteryzuje rozwój nauki od 17 wieku jest do wywalenia, to strasznie długa odpowiedź.

Reply:
Nie wiem z czego Piotrek będzie pytał na egzaminie ustnym xD ale masz rację, fiszka raczej do wyrzucenia zdroworozsądkowo